de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1232,6 +1232,112 @@ image:chapterModeling/MaturityOverview.png[Maturity overview]
+|image:chapterStateMachines/AddState.png[AddState] 
+
+|{counter:step1}.
+|After creating a transition connection in the Diagram Editor, you want to create a _TransitionTriggerEvent_ and assign it to the previously created transition. 
+
+Go in the _Editor_ using _SM_ instance in the _Navigator_: `Context` › `Open Editor` or simply by double clicking on the _SM_ instance in the _Navigator_.
+
+In the _Editor_ if you press [Add TransitionTriggerEvent] button, a new trigger event is created. 
+
+Name it under the column `Name`, and give it a description under the `detail`.
+
+|image:chapterStateMachines/Open Editor.png[Open Editor] image:chapterStateMachines/AddTransitionTriggerEvent.png[AddTransitionTriggerEvent]
+
+|{counter:step1}.
+|Now assign the created trigger event to the `Transition` you created in the diagram. Under `trigger` column in the `transitions` section, you can choose a trigger action from the list as shown in the image.
+
+NOTE: Transitions can be created or removed in the _Diagram Editor_, but also manually in the normal _Editor_.
+
+|image:chapterStateMachines/AssigningTriggers.png[AssigningTriggers] image:chapterStateMachines/OpenedWizardForTriggerActions.png[OpenedWizardForTriggerActions]
+
+|{counter:step1}.
+|After assigning, go back to the _Diagram Editor_ and update the state machine diagram using the yellow button up right the _State Machine_ block, then you will be able to see the new transition.
+|image:chapterStateMachines/UpdateStateMachine.png[UpdateStateMachine]
+
+|{counter:step1}.
+|Now having created the various components of the system and described their behavior using state machines, the _State Machine_ for the `AOCS` system can be created in a newly created _ElementConfiguration_ namely `CONOPS` (standing for concept operations). This state machine shall describe the behavior of the whole system.
+
+See Task No.4 “Creating a State Machine”
+
+|image:chapterStateMachines/AddingSystemStateMachine.png[AddingSystemStateMachine]
+
+|{counter:step1}.
+|Open the _Diagram Editor_ and add the following _States_ and _Transitions_ as shown in the image. This shall serve as an example to represent the behavior of the `AOCS` system.
+
+NOTE: The green and red arrows shown in the state machine shall represent the _AllowsConstraints_ and _ForbidsConstraints_ respectively.  For instance, the thrusters are on when the safe mode of the satellite is active to perform coarse pointing control.
+
+TIP: To use the _State Machine_ of other system components, e.g. the Camera, Reaction Wheels in Main system _State Machine_, one can just drag and drop them from the _Navigator_. 
+
+|image:chapterStateMachines/AOCSSystemStateMachine.png[AOCSSystemStateMachine] image:chapterStateMachines/AOCSTransitionsAndTriggerEvents.png[AOCSTransitionsAndTriggerEvents]
+
+|{counter:step1}.
+|For each section in _State Machine_ concept found in the Editor an excel table with the section instances can be created. This shown in the following image.

```

Comment:
"This shown in the following image" -> This is shown in the following image"

Reply:
changed